docs/releasenotes/8.0.0.rst

```diff
@@ -9,6 +9,12 @@ Python 3.5
 
 Pillow has dropped support for Python 3.5, which reached end-of-life on 2020-09-13.
 
+PyPy 7.1.x
+^^^^^^^^^^
+
+Pillow has dropped support for PyPy3 7.1.1.
+PyPy3 7.2.0, released on 2019-10-14, is now the minimum compatible version.
```

Comment:
I'm not too happy with this wording, but can't think of a way to improve this.

From https://github.com/python-pillow/Pillow/pull/4964#issuecomment-706988596:
> I suggest in general we aim to support the latest release of PyPy?
> 
> (Excluding any Python versions that are EOL by CPython, i.e. Python 2.7.)
> 
> And as Pillow 8.0.0 is a major release, it's okay to remove old compat code?

Reply:
I think it's fine, thanks!